src/main/java/model/Players.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import exception.ExceptionMessage;
+
+import java.util.List;
+
+public class Players {
+
+    private static final int MIN_PLAYER_LENGTH = 2;
+    private final List<Player> players;
+
+    public Players(List<String> players) {
+        validatePlayers(players);
+        this.players = generatePlayers(players);
+    }
+
+    public int size() {
+        return players.size();
+    }
+
+    public List<String> getPlayers() {
+        return players.stream()
+                .map(Player::getValue)
+                .toList();
+    }
+
+    private List<Player> generatePlayers(List<String> players) {
+        return players.stream()
+                .map(Player::new)
+                .toList();
+    }

```

Comment:
```suggestion
    private List<Player> generatePlayers(List<String> playerNames) {
        return playerNames.stream()
                .map(Player::new)
                .toList();
    }
```

더 명확히 써볼 수 있을 것 같아요. 😄

Reply:
말씀하신 것처럼 코드 작성 시에는 제가 제 코드를 가장 잘 알기 때문에 
"이 정도면 되겠지" 하고 넘아가곤 했는데,
협업이나 현업에서는 다른 사람들이 코드를 읽을 가능성도 있으므로
더 많은 사람들과 코드를 공유할 때를 고려해서 가독성 좋게 코드를 작성하려고 노력하겠습니다
명확한 피드백 주셔서 감사드립니다 !